courses/machine_learning/deepdive2/structured/solutions/1_explore_babyweight.ipynb

```diff
@@ -0,0 +1,591 @@
+{
```

Comment:
<ul><li> Something is missing here: "Use Cloud AI Platform Notebooks plot explorations" </li><li class="ql-indent-1">Maybe replace by "Learn how to explore a dataset by using plots in the CAIP notebooks"</li><li><strong>"#TODO</strong>&nbsp;in this student lab notebook" -&gt; "<strong>#TODO</strong>&nbsp;in this  notebook" (this is the solution not the lab, right?)</li><li>"-- try to complete this notebook first and then review the&nbsp;<a href="https://github.com/GoogleCloudPlatform/training-data-analyst/raw/867c2482916c774e183cbea40f3d30b64711aed3/courses/machine_learning/deepdive2/structured/solutions/1_explore_babyweight.ipynb" target="_blank">solution notebook</a>." -&gt; maybe remove this part since it is the solution notebook</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/665/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>Done</p>